### PR TITLE
Add JOSM

### PR DESCRIPTION
**Pull request by geant44**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/14203)

### CODE REVIEW

File: manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: OpenStreetMap.Josm
+Publisher: OpenStreetMap
+PackageName: JOSM
+PackageVersion: 17833
+License: CC-BY-SA and LGPL since April 2014
```
Comment (simon04) [inline manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml:5]: JOSM is licensed GPL-2.0-or-later, see https://josm.openstreetmap.de/browser/josm/trunk/LICENSE

File: manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Moniker: josm
+Tags:
+- map
+ShortDescription: OpenStreetMap.Josm
+Description: JOSM is an extensible editor for ​OpenStreetMap (OSM) for ​Java 8+.
+PackageUrl: https://josm.openstreetmap.de/
+Installers:
+- Architecture: x86
+  InstallerUrl: https://josm.openstreetmap.de/download/windows/josm-setup.exe
```
Comment (simon04) [inline manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml:14]: Link to specific version? https://josm.openstreetmap.de/download/windows/josm-setup-17833.exe

File: manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  InstallerSha256: 0526FEBE18A457FA7614BBFFC8B76C93F67598F45A2D7E340C7D65BF7BA52A14
+  InstallerType: exe
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+PackageLocale: en-US
+ManifestType: singleton
+ManifestVersion: 1.0.0
+
```
Comment (simon04) [inline manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml:23]: Java dependency is missing?

File: manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+PackageIdentifier: OpenStreetMap.Josm
+Publisher: OpenStreetMap
+PackageName: JOSM
+PackageVersion: 17833
```
Comment (noenandre) [inline manifests/o/​OpenStreetMap/OpenStreetMap.Josm.yaml:4]: Package version need to be a string according to the manifest singleton JSON schema.
https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.singleton.1.0.0.json

**Pull request closed without merging**